flink-tests/src/test/resources/log4j2-test.properties

```diff
@@ -28,7 +28,7 @@ appender.testlogger.name = TestLogger
 appender.testlogger.type = CONSOLE
 appender.testlogger.target = SYSTEM_ERR
 appender.testlogger.layout.type = PatternLayout
-appender.testlogger.layout.pattern = %-4r [%t] %-5p %c %x - %m%n
+appender.testlogger.layout.pattern = [%-32X{flink-job-id}] %c{0} [%t] %-5p %m%n
```

Comment:
do we need this for the test to work? I'd expect that the MDC is populated regardless of what the pattern says, but that may very well not be the case.

Reply:
No, I just wanted to add job ID to log messages in tests.
Just `%x` doesn't work, because it stands for Nested DC, while this PR uses Mapped DC.

Reply:
@rkhachatryan just to clarify: Removing the timestamp (`%-4r`) was done unintentional, correct? I will add it back in a hotfix commit.